src/Microsoft.AspNet.Mvc.DataAnnotations/DataAnnotationsClientModelValidatorProvider.cs

```diff
@@ -84,81 +85,9 @@ public void GetValidators(ClientValidatorProviderContext context)
             }
         }
 
-        private static Dictionary<Type, DataAnnotationsClientModelValidationFactory> BuildAttributeFactoriesDictionary()
+        private static Dictionary<Type, ValidationAttributeAdapterFactory> BuildAttributeFactoriesDictionary()
```

Comment:
Agree w/ @NTaylorMullen this method is now both misnamed and unnecessary. Just use `ValidationAttributeAdapterTable.AttributeFactories` directly.